selfdrive/debug/toyota_eps_factor.py

```diff
@@ -37,6 +35,7 @@ def get_eps_factor(lr, plot=False):
         torque_cmd = to_signed((m.dat[1] << 8) | m.dat[2], 16)
       elif m.address == 0x260 and m.src == 0:
         eps_torque = to_signed((m.dat[5] << 8) | m.dat[6], 16)
+        steering_pressed = abs(to_signed((m.dat[1] << 8) | m.dat[2], 16)) > STEER_THRESHOLD
```

Comment:
Is this right?

Reply:
yeah, that looks right

Reply:
awesome! do you mind running it on a route of mine? curious to know how much more accurate it is now: 14431dbeedbf3558%7C2020-11-27--13-03-23

Should be 0.88 to 0.9